core/src/main/java/org/apache/carbondata/core/cache/CacheProvider.java

```diff
@@ -135,11 +131,11 @@ public static CacheProvider getInstance() {
   private void createDictionaryCacheForGivenType(CacheType cacheType) {
```

Comment:
now it is not dictionaryCache, we can rename it.

Reply:
also can rename **dictionaryCacheAlreadyExists** API

Reply:
renamed